dashboards/individual-flows-per-country.json

```diff
@@ -3518,9 +3488,11 @@
       {
         "allValue": "*",
         "current": {
-          "selected": false,
-          "text": "All",
-          "value": "$__all"
+          "tags": [],
+          "text": "Canada",
+          "value": [
+            "Canada"
```

Comment:
I don't think we intended for Canada to be the default here, can this be changed back to All, or maybe US makes the most sense?